### PR TITLE
Update x-name-directive.md

### PR DESCRIPTION
**Pull request by JakenHerman**
## Summary

Added a tip to developers using MVVM architecture, explainin that x:Name is not a bindable directive

i.e. x:Name={Binding SetName} is not something you can do. If I am wrong, please correct me.

### CODE REVIEW

File: docs/framework/xaml-services/x-name-directive.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Uniquely identifies XAML-defined elements in a XAML namescope. XAML namescopes a
  For custom types that use .NET Framework XAML Services XAML object writers, a property that maps to `x:Name` on a type can be established or changed. You define this behavior by referencing the name of the property to map with the <xref:System.Windows.Markup.RuntimeNamePropertyAttribute> in the type definition code.  <xref:System.Windows.Markup.RuntimeNamePropertyAttribute> is a type-level attribute.  
   
  Using.NET Framework XAML Services, the backing logic for XAML namescope support can be defined in a framework-neutral way by implementing the <xref:System.Windows.Markup.INameScope> interface.  
+ 
+ The `x:Name` directive is not a bindable property. You can not set this directive via some property on a View Model if you are using MVVM architecture.
```
Comment (jzeferino) [inline docs/framework/xaml-services/x-name-directive.md]: > View Model

 should be written ViewModel.

**Review by JakenHerman — COMMENTED** [review]
Changed "View Model" to "ViewModel"

**Review by jzeferino — APPROVED** [review]
LGTM now.

**Review by SamBent — COMMENTED** [review]
This change is superfluous.  x:Name is a directive, not a property, so saying it's not a bindable property adds nothing.

**Comment by mairaw** [comment]
Thank you for your PR @JakenHerman. Given @SamBent's review, I'll be closing this issue without merging it.

**Pull request closed without merging**